marklogic-data-hub/src/main/java/com/marklogic/hub/deploy/commands/DeployDatabaseFieldCommand.java

```diff
@@ -43,6 +43,7 @@
 
     public DeployDatabaseFieldCommand() {
         setExecuteSortOrder(SortOrderConstants.DEPLOY_OTHER_DATABASES + 1);
+        setUndoSortOrder(SortOrderConstants.DELETE_OTHER_DATABASES - 1);
```

Comment:
@srinathgit This command actually shouldn't do anything on undeploy. Because it's extending AbstractResourceCommand (which simplified the implementation a great deal), it implements UndoableCommand, but the right fix is to override the "undo" method and not do anything. The command for undeploying databases will then handle undeploying.

Reply:
Also, can you submit a PR for fixing this on 5.1-develop as well?